File: Examples/CSharp/Loading-and-Saving/RetreivingFileFormat.cs
using System.IO;
using Aspose.Note;
using Aspose.Note.Saving;
using System;
namespace Aspose.Note.Examples.CSharp.Loading_Saving
{
    public class RetreivingFileFormat
    {
        public static void Run()
        {
            // ExStart:RetreivingFileFormat
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            var document = new Aspose.Note.Document(dataDir + "Aspose.one");
            //switch (document.FileFormat)
            //{
            //    case FileFormat.OneNote2010:
            //        // process OneNote 2010
            //        break;
            //    case FileFormat.OneNoteOnline:
            //        // process OneNote Online
            //        break;
            //}
            // ExEnd:RetreivingFileFormat           
        }
    }
}

File: Examples/VisualBasic/Loading-and-Saving/RetreivingFileFormat.vb
Imports System.IO
Imports  Aspose.Note
Imports Aspose.Note.Saving

Namespace CSharp.Loading_Saving
    Public Class RetreivingFileFormat
        Public Shared Sub Run()
            ' ExStart:RetreivingFileFormat
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadingAndSaving()

            Dim document = New Document(dataDir & Convert.ToString("Aspose.one"))
            Select Case document.FileFormat
                Case FileFormat.OneNote2010
                    ' process OneNote 2010
                    Exit Select
                Case FileFormat.OneNoteOnline
                    ' process OneNote Online
                    Exit Select
            End Select
            ' ExEnd:RetreivingFileFormat           
        End Sub
    End Class
End Namespace